lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/experianKbvCri/ContractTest.java

```diff
@@ -0,0 +1,1041 @@
+            "eyJpc3MiOiJpcHYtY29yZSIsInN1YiI6Imlwdi1jb3JlIiwiYXVkIjoiZHVtbXlFeHBlcmlhbktidkNvbXBvbmVudElkIiwiZXhwIjo0MDcwOTA5NzAwLCJqdGkiOiJTY25GNGRHWHRoWllYU181azg1T2JFb1NVMDRXLUgzcWFfcDZucHYyWlVZIn0";
+    // Signature generated using JWT.io
+    private static final String CLIENT_ASSERTION_SIGNATURE =
+            "aJOEpvnBRpaptv_2T7L5aCzhTdvlNaGNh3uwuK1f5cC9he9izuIr60s2_Y6-DIPEWLE0_L6ckgdIsy9G7yj8jA";
+    // We hardcode the VC headers and bodies like this so that it is easy to update them from JSON
+    // sent by the CRI team
+    private static final String VALID_VC_HEADER =
+            """
+            {
+              "alg": "ES256",
+              "typ": "JWT"
+            }
+            """;
+
+    private static final String VALID_VC_BODY =
+            """
+                {
+                  "iss": "dummyExperianKbvComponentId",
+                  "sub": "test-subject",
+                  "nbf": 4070908800,
+                  "vc": {
+                    "type": [
+                      "VerifiableCredential",
+                      "IdentityCheckCredential"
+                    ],
+                    "credentialSubject": {
+                      "name": [
+                        {
+                          "nameParts": [
+                            {
+                              "type": "GivenName",
+                              "value": "Mary"
+                            },
+                            {
+                              "type": "FamilyName",
+                              "value": "Watson"
+                            }
+                          ]
+                        }
+                      ],
+                      "birthDate": [
+                        {
+                          "value": "1932-02-25"
+                        }
+                      ],
+                      "address": [
+                        {
+                          "uprn": "10022812929",
+                          "organisationName": "FINCH GROUP",
+                          "subBuildingName": "UNIT 2B",
+                          "buildingNumber": "16",
+                          "buildingName": "COY POND BUSINESS PARK",
+                          "dependentStreetName": "KINGS PARK",
+                          "streetName": "BIG STREET",
+                          "doubleDependentAddressLocality": "SOME DISTRICT",
+                          "dependentAddressLocality": "LONG EATON",
+                          "addressLocality": "GREAT MISSENDEN",
+                          "postalCode": "HP16 0AL",
+                          "addressCountry": "GB"
+                        }
+                      ]
+                    },
+                "evidence": [
+                    {
+                      "checkDetails": [
+                        {
+                          "checkMethod": "kbv",
+                          "kbvQuality": 2,
+                          "kbvResponseMode": "free_text"
```

Comment:
Not sure where this has come from but the Experian KBV qs are all multiple choice

Reply:
Looks like it came from here: https://github.com/govuk-one-login/architecture/blob/main/rfc/0024-identity-vc-for-checks.md#knowledge-based-verification-kbv-check which where the Orange team pointed Roger.
I've had a quick search in Confluence and can't see any more specific Experian example VCs

Reply:
Yeah the RFC is just describing a generic KBV CRI. I'd probably change it to multiple choice for now and can be something to come back to if we need to?